src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetManagedTemplatePackage.cs

```diff
@@ -156,6 +112,24 @@ public DateTime LastChangeTime
             }
         }
 
+        public string? Trusted
+        {
+            get => Details.TryGetValue(TrustedKey, out string trusted) ? trusted : false.ToString();
+            set => Details.UpdateOrRemoveValue(TrustedKey, value!, InsertionCondition);
```

Comment:
here (and below) is the same issue with nullable value - if null is passed, exception will be thrown.
```suggestion
            set => Details.UpdateOrRemoveValue(TrustedKey, value, RemovalCondition);
```